### PR TITLE
Add `available_values` property to dashboard template variables

### PR DESCRIPTION
**Pull request by adil-alimbetov**
This PR adds the `available_values`  property to template variables.
**Related PRs:**
- https://github.com/DataDog/datadog-api-spec/pull/1133
- https://github.com/DataDog/dogweb/pull/63689

### CODE REVIEW

**Comment by adil-alimbetov** [comment]
Not sure if I should remove available_values from examples for test to pass 🤔

File: docs/resources/dashboard.md [hunk 1]
```diff
@@ -11013,6 +11013,7 @@ Required:
 
 Optional:
 
+- **available_values** (List of String) The list of values that the template variable drop-down is be limited to
```
Comment (apigirl) [inline docs/resources/dashboard.md:11016]:
```suggestion
- **available_values** (List of strings) The list of values that the template variable drop-down is be limited to.
```

**Review by apigirl — COMMENTED** [review]
one small docs feedback, otherwise lgtm

File: docs/resources/dashboard.md (hunk 1 above)
Comment (adil-alimbetov) [reply docs/resources/dashboard.md:11016]: I'm afraid this one was autogenerated and beyond my control 😅

**Review by DrkSephy — APPROVED** [review]

**Review by therve — APPROVED** [review]

**Review by therve — APPROVED** [review]

**Pull request merged**